src/components/App/App.jsx

```diff
@@ -6,19 +6,21 @@ import { useYoutubeVideos } from '../../utils/hooks/useYoutubeVideos';
 // views
 import Home from '../../pages/Home/HomePage';
 import DetailsPage from '../../pages/Details';
+import { useGlobalContext } from '../../providers/GlobalProvider/GlobalProvider';
+import { ThemeProvider } from 'styled-components';
 
 const App = () => {
-  const [search, setSearch] = useState('');
   const [page, setPage] = useState({
     home: true,
     video: null,
   });
```

Comment:
I know this went through since the past deliverable but I think instead of having a `useState` with two different properties it is better if this was refactored into something like 

```js
// this const in a file inside /utils folder
const PAGES = {
   home = 'home',
   video = 'video',
}
[...]
const [currentPage, setCurrentPage] = useState(PAGES.home);
// and where we will call setCurrentPage in the future
setCurrentPage(PAGES.video);
```